components/ide/jetbrains/backend-plugin/gradle-stable-rider.properties

```diff
@@ -0,0 +1,13 @@
+# this file is auto generated by components/ide/gha-update-image/index-jb.ts
+# See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
+# for insight into build numbers and IntelliJ Platform versions.
+pluginSinceBuild=241.17890
```

Comment:
Can you share how I learn more about build number 241.17890? I ask because I want to assert it came from an expected build.

Reply:
JetBrains' official document is not clear enough, let's ignore it now. because I'm confused with buildNumber as well -> it said buildNumber can be `platformVersion` but when building, it will complain not found.

`pluginSinceBuild=241.17890` comes from `gradle-stable.properties`

Reply:
There're two links could include what we want.
- https://www.jetbrains.com/intellij-repository/snapshots
- https://www.jetbrains.com/updates/updates.xml

`pluginSinceBuild` the range of buildNumber that current plugin is compatible with, in my opinion, once it's small enough to cover Rider 2024.1 to 2024.1.4, then we're good to go. And this is a part of `How to Test` section **❗ Oldest pinnable Rider is working with current build of backend plugin**

Reply:
> pluginSinceBuild=241.17890

Comes from backend plugin: /workspace/gitpod/components/ide/jetbrains/backend-plugin/gradle-stable.properties

There is also /workspace/gitpod/components/ide/jetbrains/gateway-plugin/gradle-stable.properties, but I assume that is not relevant for this particular case?

Reply:
Yes, comes from `backend-plugin/gradle-stable.properties`, Rider can't start up because our backend-plugin is incompatible with Rider, so we need to update bakcend-plugin.

- `backend-plugin`: Inside workspace, will loaded by JetBrains IDE, do things like heartbeat, metrics ...
- `gateway-plugin`: Installed on users's Desktop (JetBrains Gateway), to connect to workspace